packages/infra/bin/infrastructure.ts

```diff
@@ -77,8 +78,26 @@ async function deploy(config: EnvConfig) {
         description: `VPN Configuration for routing HL7 messages from ${hieConfig.name}`,
       });
 
+      /**
+       * We add explicit dependencies here because the VPN tunnel infra is booted up _outside_ of the standard
+       * develop / master CI pipelines to make the integration process easier + prevent needing to ship changes
+       * back forth through CI in order to get tunnels, networking rules, etc. deployed.
+       */
+      vpnStack.addDependency(secretsStack);
```

Comment:
This was missing and has recently caused some issues when trying to deploy the vpn stack via our usual methods, adding it in here.